app/src/store/user.ts

```diff
@@ -102,7 +102,7 @@ export const logout = createAsyncThunk<{ accessToken: string; email: string }>(
       await auth0.credentialsManager.clearCredentials();
       return { accessToken: '', email: '' };
     } catch (error) {
-      return rejectWithValue('LOGOUT_ERROR');
+      return rejectWithValue(error);
```

Comment:
No at the time this is just a subjective message I put for this error. But since auth0 will return an error if any of the async calls fail, I figure it would be best to return their error.